--- archive/asm-xml.bnd	2012-10-12 20:56:44.000000000 +0200
+++ archive/asm-xml.bnd-gil	2012-10-19 18:14:41.534123148 +0200
@@ -32,7 +32,6 @@
 Implementation-Title:  ASM XML classes
 Implementation-Version: ${product.artifact}
 Implementation-Vendor: Eugene Kuleshov
-Class-Path: asm-${product.artifact}.jar asm-util-${product.artifact}.jar asm-attrs-${product.artifact}.jar
 Main-Class: org.objectweb.asm.xml.Processor
 
 Bundle-Name: ASM xml classes
@@ -46,7 +45,8 @@
 
 Import-Package: \
   javax.xml.transform.*;resolution:="optional",\
-  org.xml.sax.*;resolution:="optional"
+  org.xml.sax.*;resolution:="optional",\
+  org.objectweb.asm.*;version:="[${product.artifact},${product.artifact}]"
 
 Bundle-RequiredExecutionEnvironment: J2SE-1.3
 
